internal/export/distro/zeromq.go

```diff
@@ -27,8 +23,7 @@ func initZmq(eventCh chan *models.Event) {
 	defer q.Close()
 
 	LoggingClient.Info("Connecting to zmq...")
-	url := fmt.Sprintf("tcp://%s:%d", Configuration.DataHost, zeroMQPort)
-	q.Connect(url)
+	q.Connect(Configuration.MessageQueue.Uri())
```

Comment:
@jpwhitemn @drasko Calling your attention to this file. Resolution for https://github.com/edgexfoundry/edgex-go/issues/612

Reply:
thanks @tsconn23